cmd/run.go

```diff
@@ -567,6 +567,19 @@ func executeRun(runFilePath string, apps []runfileconfig.App) (bool, error) {
 	return exitWithError, closeError
 }
 
+func getLogWriter(app runfileconfig.App, fileLogWriterCloser io.WriteCloser, logDestination runfileconfig.LogDestType) io.Writer {
```

Comment:
I would like to ask why the `app` is needed

Reply:
removed